interfaces/builtin/common.go

```diff
@@ -123,3 +126,31 @@ func (iface *commonInterface) ConnectedSlotSnippet(plug *interfaces.Plug, slot *
 func (iface *commonInterface) AutoConnect(*interfaces.Plug, *interfaces.Slot) bool {
 	return !iface.rejectAutoConnectPairs
 }
+
+func (iface *commonInterface) KModConnectedPlug(spec *kmod.Specification, plug *interfaces.Plug, slot *interfaces.Slot) error {
+	for _, m := range iface.connectedPlugKModModules {
+		spec.AddModule(m)
```

Comment:
While we don't error in practice in the current code I think we want to check for errors here.